sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -92,6 +92,7 @@ int _handle_reply(ng_pktsnip_t *pkt, uint64_t time)
         return 0;
```

Comment:
in l74, too. (Or you don't release the packet here, but after the call to `_handle_reply()` ;-))